app/components/oral_history/admin/combined_audio_derivatives_component.html.erb

```diff
@@ -8,19 +8,19 @@
             The combined audio derivatives, created from <%= work_available_members_count %> published audio segments, are up to date. Their fingerprint is <code class="border"><%= combined_audio_fingerprint%></code>.
         </p>
         <ul>
-          <% if combined_mp3_audio.present? %>
+          <% if combined_m4a_audio.present? %>
             <li>
-              Use the following mp3 link as the OHMS Media URL:<br/>
+              Use the following M4A link as the OHMS Media URL:<br/>
               <small>
-                <%= link_to combined_mp3_audio,  combined_mp3_audio %>
+                <%= link_to combined_m4a_audio,  combined_m4a_audio %>
               </small>
             </li>
           <% end %>
-          <% if combined_webm_audio.present? %>
-            <li>
-              The corresponding webm link is provided for reference only:<br/>
+          <% if combined_mp3_audio.present? # REMOVE_AFTER_MP3_TO_M4A_MIGRATION %>
+            <li class="alert-danger">
+              The following MP3 link will be deleted soon. Click "Generate combined audio derivatives below" to create an M4A; use that instead.<br/>
               <small>
-                <%= link_to combined_webm_audio, combined_webm_audio %>
+                <%= link_to combined_mp3_audio,  combined_mp3_audio %>
               </small>
             </li>
           <% end %>
```

Comment:
OK, sorry, I thought we'd show the mp3 only if the m4a did NOT exist, and the mp3 did -- cause that's a thing that can happen while we are in intermediate state, right?